pkgs/tools/networking/opensnitch/ui.nix

```diff
@@ -33,6 +34,7 @@ python3Packages.buildPythonApplication rec {
     pyinotify
     notify2
     pyasn
+    qtwayland
```

Comment:
Please add it to `buildInputs`, `wrapQtAppsHook` will pick it up from there so it's unnecessary to propagatedBuildInputs it.

Reply:
Done

Reply:
You forgot to push

Reply:
Oof, sorry.

Reply:
It looks like you fixed it up in the meantime so I'll leave this be?